### PR TITLE
Epic 1 Yankiv Olesia

### PR DESCRIPTION
**Pull request by OlesiaYankiv**

### CODE REVIEW

File: ai_12/olesia_yankiv/epic 1/bank.cpp
Comment (yanalebedynska) [inline ai_12/olesia_yankiv/epic 1/bank.cpp:1]: Nice code!

**Review by yanalebedynska — COMMENTED** [review]
Все добре

**Review by VoroninOlexandr — COMMENTED** [review]
The best code i have ever seen

**Review by rtriska — APPROVED** [review]

**Review by yanalebedynska — COMMENTED** [review]
Чудово зроблено

**Pull request merged**